### ISSUE TITLE
Discogs script only partially working

### ISSUE THREAD

**Issue by morrinson** [murdos/musicbrainz-userscripts#413]
The script fails to open the import button in the sidebar, but all the other buttons / links (for label, artist, etc. redirections to MusicBrainz) are properly showing.
I've tried basically everything, log out and back in, deactivate ad blocker, change language, manually update and reinstall script, but the same behavior happens on three different computers (all with Firefox / Windows).

![unknown](https://user-images.githubusercontent.com/60886332/130211090-7ad16def-fcd5-4252-bf80-461d2ba7f658.png)

**Comment by liftarn** [comment]
I got this.
![bild](https://user-images.githubusercontent.com/4818861/131477266-2ca54173-1f87-4bc6-86ca-f25896de00b2.png)

**Comment by morrinson** [comment]
Same error message on my side with respect to this release.
Regarding the original issue of this thread, it seems to have been corrected since. I leave the issue open just in case it is helpful for the issue mentioned by liftarn.

**Comment by lyz-code** [comment]
@morrinson were you able to make it work?

**Comment by morrinson** [comment]
@lyz-code Yeah it finally worked (after a fix around 31 Aug I guess). But it was not for a master, which indeed cannot be imported. The original post's screenshot was related to this page: https://www.discogs.com/release/15672190-Chui-Wan-Eye
